autocompleter/utils.py

```diff
@@ -185,3 +185,45 @@ def term_phrase_already_aliased(aliasable_phrase_start, aliasable_phrase_end,
         elif aliasable_phrase_end <= aliased_phrase_end:
             return True
     return False
+
+
+def validate_facets(facets):
```

Comment:
since this is used once and only once and I can't right now imagine a situation where it would be used elsewhere, I would make this a staticmethod of SuggestView called _validate_facets